libbeat/docs/shared-faq.asciidoc

```diff
@@ -144,3 +144,32 @@ endif::no-output-logstash[]
 === Monitoring UI shows fewer Beats than expected
 
 If you are running multiple Beat instances on the same host, make sure they each have a distinct `path.data` value.
+
+ifndef::no_dashboards[]
+[[could-not-locate-index-pattern]]
+=== Dashboard could not locate the index-pattern
+
+Typically {beatname_uc} sets up the index pattern automatically when it
+loads the index template. However, if for some reason {beatname_uc} loads the
+index template, but the index pattern does not get created correctly, you'll see
+a "could not locate that index-pattern" error. To resolve this problem:
+
+. Try running the `setup` command again. For example: +./{beatname_lc} setup+.
+
+. If that doesn't work, go to the Management app in {kib}, and under
+*Index Patterns*, look for the pattern.
+.. If the pattern exists, click it, then click Refresh.
+
+.. If the pattern doesn't exist, create it manually.
++
+--
+* Set the *Time filter field name* to `@timestamp`.
+* If you're using a custom index name, set the *Custom index pattern ID*
```

Comment:
I think that this is needed in any case, at least in some versions of Kibana. If not the index is created with a randomized pattern ID.